frontend/elements/src/entry/admin/locale/landing.ts

```diff
@@ -0,0 +1,206 @@
+                right: 10px;
+            }
+            :host([saving]) .saving-indicator::after {
+                padding: 10px;
+                content: 'Saving...';
+            }
+
+            .icon-button {
+                height: 33px;
+                display: flex;
+                align-items: center;
+                column-gap: 10px;
+            }
+            .icon-button button {
+                background-color: transparent;
+                border: 0;
+                cursor: pointer;
+                padding: 0;
+                width: 50px;
+            }
+            .icon-button button img {
+                max-height: 100%;
+                max-width: 50px;
+            }
+            .icon-button.select-columns {
+                grid-column: 3;
+            }
+            .icon-button.add-text {
+                grid-column: 4;
+            }
+
+            .table {
+                grid-column: 1 / -1;
+                display: grid;
+                grid-template-columns: repeat(11, auto);
+            }
+            ::slotted([slot=rows]) {
+                display: contents;
+            }
+            :host([sortOrder=asc]) {
+                --sort-arrow: '⇩';
+            }
+            :host([sortOrder=desc]) {
+                --sort-arrow: '⇧';
+            }
+
+            dialog {
+                border-radius: 8px;
+                border: solid #00000040 1px;
+            }
+            dialog::backdrop {
+                background-color: #00000060;
+            }
+            .column-selection-contents {
+                display: grid;
+                grid-template-columns: repeat(2, 200px);
+            }
+            .column-selection-contents hr {
+                grid-column: 1 / -1;
+                width: 100%;
+            }
+            .column-selection-contents header {
+                grid-column: 1 / -1;
+                text-align: center;
+            }
+            .column-selection-contents ul {
+                list-style: none;
+                padding: 0;
+                margin: 0;
+            }
+            .column-selection-contents .actions {
+                grid-column: 1 / -1;
+                display: flex;
+                justify-content: flex-end;
+                column-gap: 10px;
+            }
+        `];
+    }
+
+    @property({type: Array})
+    public bundles: Bundle[] = [];
+
+    @property({type: Array})
+    public columns: Column[] = []
+
+    @property({type: Boolean, reflect: true})
+    public saving = false;
+
+    @property({reflect: true})
+    public sortOrder: 'asc' | 'desc' = 'asc';
+
+    @query('dialog', true)
+    public dialog!: HTMLDialogElement;
+
+    private onBundleSelect(e: Event) {
+        const select = e.target as HTMLSelectElement;
+        const options = select.options;
+        this.bundles.forEach((b, i) => {
+            b[1] = options[i].selected;
+        });
+        
+        this.dispatchEvent(
```

Comment:
see comment below - basically, it's better to slot the bundles in and avoid the CustomEvent here